src/Routes.js

```diff
@@ -23,14 +24,20 @@ import Programs from "./organizations/Programs";
 import About from "./home/About";
 import Benefit from "./home/Benefit"
 import Nav from "./Navigation"
-import Sidebar from "./sidebar/Sidebar"
+import Sidebar from "./sidebar/Sidebar";
```

Comment:
Hey @arkumari2000,  can you please remove this line cause Sidebar has already been declared :)